OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -2198,6 +2198,52 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 						node.Key = "-WithSpriteRotorOverlay";
 				}
 
+				if (engineVersion < 20151005)
+				{
+					// Move certain properties from Parachutable to new WithParachute trait
+					// Add dependency traits to actors implementing Parachutable
+					// Make otherwise targetable parachuting actors untargetable
```

Comment:
This comment seems to be unrelated copy & pasta from https://github.com/matija-hustic/OpenRA/blob/69696fc154ebdf596cbb77c438f4f091b552d2d5/OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs#L1569